DependencyInjection/Compiler/SystemInfoCollectorPass.php

```diff
@@ -37,4 +44,38 @@ public function process(ContainerBuilder $container)
         $infoCollectorRegistryDef = $container->findDefinition('support_tools.system_info.collector_registry');
         $infoCollectorRegistryDef->setArguments([$infoCollectors]);
     }
+
+    private function processSystemInfo(ContainerBuilder $container)
+    {
+        if (!$container->hasParameter('ez_support_tools.powered_by_options.enabled') ||
```

Comment:
While `ez_support_tools` is ok-ish[1] for semantic configuration - since it conforms to Symfony fallback, I wouldn't use parameter name-wise detached from eZ Platform. It gives wrong impression of separation. Support tools in current version are meant for eZ Platform only

Rather something like:
`ezplatform.support_tools.*`
or
`ezplatform.system_info.*` - while might be a bit confusing with regards to the name of the package itself, closer to its function.

---
[1] see the next diff comment

Reply:
Ok, yeah indeed that is what Sylvain proposed initially with:
```
ezplatform:
  system_info:
    http_header:
      expose_product_name: true
      expose_product_version: false
```

So what about:
```
# Default configuration for extension with alias: "ezplatform_system_info"
ezplatform:
  system_info:
    # Control if system should generate a powered by header to promote eZ Platform usage
    powered_by:
        # Activates powered by header
        enabled:              true
        # Sets how much of version info is exposed in powered by. Examples: major: 2, minor: 2.5
        release:              minor # One of "major"; "minor"; null
        # Optional config to override the powered by naming determined by the system
        custom_name:          ''
```

Reply:
The above is fine with me. Though maybe distinguish between `null` and empty string for `custom_name`, with the former indicating it's not an override, but not set, so eZ Platform can be set. Someone might want to override with an empty string ;)